### ISSUE TITLE
Gettext adapter constructor error with Phalcon 2.0.10

### ISSUE THREAD

**Issue by Gamblt** [phalcon/cphalcon#11429]
After updating on 2.0.10 I've got fatal error in Phalcon gettext adapter constructor.

```
[error] 7465#7465: *2696 FastCGI sent in stderr: "PHP message: PHP Warning:  Invalid callback MyProject\Components\Translate::prepareoptions, cannot access private method MyProject\Components\Translate::prepareOptions() in /var/www/MyProject/app/components/Translate.php on line 39
PHP message: PHP Stack trace:
PHP message: PHP   1. {main}() /var/www/MyProject/public/index.php:0
PHP message: PHP   2. MyProject\Bootstrap->run() /var/www/MyProject/public/index.php:15
PHP message: PHP   3. MyProject\Components\Translate->__construct() /var/www/MyProject/app/Bootstrap.php:162
PHP message: PHP   4. Phalcon\Translate\Adapter\Gettext->__construct() /var/www/MyProject/app/components/Translate.php:39
PHP message: PHP Fatal error:  Uncaught exception 'RuntimeException' with message 'Call to undefined method MyProject\Components\Translate::prepareoptions(), did you mean 'prepareOptions'?' in /var/www/MyProject/app/components/Translate.php:39
Stack trace:
#0 /var/www/MyProject/app/components/Translate.php(39): Phalcon\Translate\Adapter\Gettext->__construct(Array)
#1 /var/www/MyProject/app/Bootstrap.php(162): MyProject\Components\Translate->__construct(Object(Phalcon\Config))
#2 /var/www/MyProject/public/index.php(15): MyProject\Bootstrap->run()
#3 {main}
  thrown in /var/www/MyProject/app/components/Translate.php on line 39" while reading response header from upstream, client: 127.0.0.1
```

2.0.9 works good.

**Comment by sergeyklay** [comment]
Thanks for reporting. Wilд be fixed quickly

**Comment by sergeyklay** [comment]
Fixed in `2.0.x`

**Comment by Gamblt** [comment]
The same issue in 2.1.x branch
`[error] 6164#6164: *3 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Call to private method Phalcon\Translate\Adapter\Gettext::getoptionsdefault() from context 'MyProject\Components\Translate' in /home/gambit/repos/MyProject/app/components/Translate.php:51
Stack trace:
#0 [internal function]: Phalcon\Translate\Adapter\Gettext->prepareOptions(Array)
#1 /home/gambit/repos/MyProject/app/components/Translate.php(51): Phalcon\Translate\Adapter\Gettext->__construct(Array)
#2 /home/gambit/repos/MyProject/app/Bootstrap.php(158): MyProject\Components\Translate->__construct(Object(Phalcon\Config))
#3 /home/gambit/repos/MyProject/public/index.php(15): MyProject\Bootstrap->run()
#4 {main}`
